src/main/java/com/example/cinematicketbookingapp/mapper/SeatDtoMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.example.cinematicketbookingapp.mapper;
+
+import com.example.cinematicketbookingapp.dto.SeatListDto;
+import com.example.cinematicketbookingapp.model.Seat;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Czy to faktycznie jest service?

Reply:
No właśnie nie jest i w sumie mogła by być ogólna adnotacja `@Component`, ale pamiętam, że kiedyś zapytałem na jakimś forum jakie adnotacje Spring Beans stosować do klas mapperów i uzyskałem odpowiedź, że może być `@Service`, bo mapper również świadczy nam jakieś usługi – i tak mi już zostało. Oczywiście rozumiem sugestię i zmieniam podejście 😊.

Reply:
Jak dla mnie to zwykły skromny komponencik, `@Service` raczej rezerwuję do klas, które stworzonę są do wielkich rzeczy (czyt. logiki biznesowej).